index.html

```diff
@@ -129,7 +129,61 @@ <h2 class="ui divider horizontal">Ils l'utilisent déjà</h2>
         'contract'
       ],
       searchFullText: true,
-      source: data
+      source: data,
+      templates: {
+        message: function(type, message) {
+          html = '<p class="searchError">Aucune API ne correspond à votre recherche</p>'
+          + '<a href="' + action.url + '" class="action">'
+          +   action.text
+          + '</a>';
+          return html;
+        },
+        standard: function(response, fields) {
+          var
+            html = ''
+          ;
+          if(response[fields.results] !== undefined) {
```

Comment:
You don't need this. You're iterating over something. If the iteratee is empty, nothing will happen.